extensions/amp-carousel/0.2/amp-carousel.js

```diff
@@ -644,9 +644,16 @@ class AmpCarousel extends AMP.BaseElement {
     if (!this.isViewerEmbeded_) {
       return;
     }
-    listen(this.scrollContainer_, 'touchmove', (event) => {
-      event.shouldViewerCancelPropagation = true;
-    });
+    listen(
+      this.scrollContainer_,
+      'touchmove',
+      (event) => {
+        event.shouldViewerCancelPropagation = true;
+      },
+      {
```

Comment:
I'm not certain we want this to be passive.

Looking at the rest of the review first and will circle back.